### PR TITLE
chore: use GOOGLE_CLOUD_PROJECT and update reads/filters region tags

### PR DESCRIPTION
**Pull request by busunkim96**
I left this in limbo for too long and new commits were added to python-docs-samples. These are the contents of https://github.com/GoogleCloudPlatform/python-docs-samples/pull/4065 and https://github.com/GoogleCloudPlatform/python-docs-samples/pull/4022

See https://github.com/GoogleCloudPlatform/python-docs-samples/commits/master/bigtable

### CODE REVIEW

**Comment by kolea2** [comment]
@googlebot

File: samples/quickstart/noxfile.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def get_pytest_env_vars():
     env_key = TEST_CONFIG['gcloud_project_env']
     # This should error out if not set.
     ret['GOOGLE_CLOUD_PROJECT'] = os.environ[env_key]
-    ret['GCLOUD_PROJECT'] = os.environ[env_key]
+    ret['GOOGLE_CLOUD_PROJECT'] = os.environ[env_key]
```
Comment (kolea2) [inline samples/quickstart/noxfile.py:75]: can we delete one of these lines?

Comment (kolea2) [reply samples/quickstart/noxfile.py:75]: (there's a few other examples of this in other files as well)

Comment (busunkim96) [reply samples/quickstart/noxfile.py]: Whoops, bad copy-pasta 😅 . Removed the duplicates

Comment (kolea2) [reply samples/quickstart/noxfile.py]: 😄  thank you!

**Pull request merged**